mirrord/operator/src/client.rs

```diff
@@ -461,28 +446,18 @@ where
     where
         P: Progress,
     {
-        match Version::parse(&self.operator.spec.operator_version) {
-            Ok(operator_version) => {
-                let mirrord_version = Version::parse(env!("CARGO_PKG_VERSION")).unwrap();
-
-                if operator_version > mirrord_version {
-                    let message = format!(
-                        "mirrord binary version {} does not match the operator version {}. Consider updating your mirrord binary.",
-                        mirrord_version,
-                        operator_version
-                    );
-                    progress.warning(&message);
-                    false
-                } else {
-                    true
-                }
-            }
-
-            Err(error) => {
-                tracing::debug!(%error, "failed to parse operator version");
-                progress.warning("Failed to parse operator version.");
-                false
-            }
+        let mirrord_version = Version::parse(env!("CARGO_PKG_VERSION")).unwrap();
```

Comment:
```suggestion
        let mirrord_version = Version::parse(env!("CARGO_PKG_VERSION")).expect("Something went wrong when parsing mirrord version!");
```

Let's change `unwraps` to be a bit less scary/give more context to the user.